hasjob/views/listing.py

```diff
@@ -886,36 +886,7 @@ def editjob(hashid, key, domain=None, form=None, validated=False, newpost=None):
     elif request.method == 'POST':
         flash("Please review the indicated issues", category='interactive')
     elif request.method == 'GET':
-        # Populate form from model
-        form.job_headline.data = post.headline
-        form.job_headlineb.data = post.headlineb
-        form.job_type.data = post.type_id
-        form.job_category.data = post.category_id
-        form.job_location.data = post.location
-        form.job_relocation_assist.data = post.relocation_assist
-        form.job_description.data = post.description
-        form.job_perks.data = True if post.perks else False
-        form.job_perks_description.data = post.perks
-        form.job_how_to_apply.data = post.how_to_apply
-        form.company_name.data = post.company_name
-        form.company_url.data = post.company_url
-        # form.poster_name.data = post.fullname  # Deprecated 2013-11-20
-        form.poster_email.data = post.email
-        form.twitter.data = post.twitter
-        form.hr_contact.data = int(post.hr_contact or False)
-        form.collaborators.data = post.admins
-
-        form.job_pay_type.data = post.pay_type
-        if post.pay_type is None:
-            # This kludge required because WTForms doesn't know how to handle None in forms
-            form.job_pay_type.data = -1
-        form.job_pay_currency.data = post.pay_currency
-        form.job_pay_cash_min.data = post.pay_cash_min
-        form.job_pay_cash_max.data = post.pay_cash_max
-        form.job_pay_equity.data = bool(post.pay_equity_min and post.pay_equity_max)
-        form.job_pay_equity_min.data = post.pay_equity_min
-        form.job_pay_equity_max.data = post.pay_equity_max
-
+        form.populate(post)
```

Comment:
I think this method should be called `populate_from` since there is also a `populate_obj` method.